app/views/admin/visit_requests/index.slim

```diff
@@ -1,33 +1,37 @@
-h2 = event.title
-= event_status_label(event)
+.ui.stackable.grid
```

Comment:
when we do changes in view - should we add screenshots ? what do you think @kushniryb ?